src/domain/Footer/Footer.tsx

```diff
@@ -41,6 +41,13 @@ const Footer: React.FC<FooterProps> = ({ className }) => {
         >
           {t('footer.sea_rescue')}
         </a>
+        <a
+          className={styles.link}
+          href={t('footer.feedback_url')}
+          target={'_blank'}
```

Comment:
https://searchenginelaws.com/seo/what-is-rel-noopener-noreferrer-tag/
```suggestion
          target="_blank"
          rel="noopener noreferrer"
```

Reply:
Done.

Reply:
These links are still missing `rel="noopener noreferrer`

Reply:
Good catch. Also added these to `CardLinks` which was still missing them.